Week-3/Assignment 4/script.js

```diff
@@ -0,0 +1,19 @@
+function createStack() {
```

Comment:
The solution given is simple yet efficient. Also try implementing the solution using closures.